altair/vegalite/v5/api.py

```diff
@@ -350,7 +350,7 @@ def __init__(
 
     @utils.deprecated(version="5.0.0", alternative="to_dict")
     def ref(self) -> dict:
-        """'ref' is deprecated. No need to call '.ref()' anymore."""
+        """'ref' is deprecated. Use '.to_dict()' instead."""
```

Comment:
Hm, not sure about this. See related issue https://github.com/vega/altair/issues/3059 and PR https://github.com/vega/altair/pull/3063 why it was stating that there is no need to call `.ref()` anymore.

Reply:
> Hm, not sure about this. See related issue #3059 and PR #3063 why it was stating that there is no need to call `.ref()` anymore.

Thanks for linking those @mattijn 

I'm thinking both the docstring and the deprecation should be updated.

Reply:
I thought I'd push with the original message in [`23a8eb5` (#3509)](https://github.com/vega/altair/pull/3509/commits/23a8eb5babe9311d093134e9642a746efe78629b) and if you'd like to add context I'll accept the suggestion(s)